scripts/runalltests.sh

```diff
@@ -0,0 +1,37 @@
+workdir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"/testslog
+testdir=$workdir/../../../tests
+
+if [ -d $workdir ]; then
+rm -r $workdir
+fi
+mkdir $workdir
+exec &> $workdir/testlog.txt
+cd $testdir
+
+echo "Fetching the test repository:"
+git pull
+echo "Test Repository last commit hash:"
+git rev-parse HEAD
+cd ..
```

Comment:
So this script somehow assumes that it is run from within a different repository? What if we change this to just use `git clone github.com/...` instead of relying on being part of a certain repository that is already checked out? If we clone it (we can do it with `--depth 1` to save bandwidth and time), we can always run it everywhere and even on a different server if needed, as long as it supports sending mail.
If we do that, then we would need
`workdir="$(mktemp -d)"` - which creates a new temporary directory.